tests/telemetry/test_telemetry.py

```diff
@@ -60,3 +76,25 @@ def test_telemetry_enabledbydefault(duthost):
         if str(k) == "status":
             status_expected = "enabled";
             pytest_assert(str(v) == status_expected, "Telemetry feature is not enabled")
+
+def test_telemetry_gnmi_get(duthost, ptfhost):
+    """Run gnmi_get from ptfdocker and verify data streaming
+    """
+    certs_del_status = duthost.shell('/usr/bin/redis-cli -n 4 del "TELEMETRY|certs"', module_ignore_errors=False)['stdout_lines']
+    set_client_auth = duthost.shell('/usr/bin/redis-cli -n 4 hset "TELEMETRY|gnmi" "client_auth" "false"', module_ignore_errors=False)['stdout_lines']
+    logger.info('start telemetry testing')
+    restart_status = restart_telemetry(duthost)
+    if restart_status == bool("true"):
+        logger.info('telemetry process restarted. Now run gnmi_get on ptfdocker')
+        #Now run gnmi_get on ptfdocker
+        dut_ip = duthost.setup()['ansible_facts']['ansible_eth0']['ipv4']['address']    
+        cmd = '''
+              cd ~/etc/go/bin &&
+             ./gnmi_get -xpath_target COUNTERS_DB COUNTERS/Ethernet0 -target_addr {0}:8080 -insecure
```

Comment:
After the telemetry agent restarts, what port is it using? I notice that "'50051" should be in config still, why is it not being used?

Reply:
what about secure mode? are we testing the secure mode?

Reply:
@hui-ma If config entries for certs are not present it starts on 8080 port. here is the code for reference  https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-sonic-telemetry/telemetry.sh

Reply:
@lguohan We have verified agent comes up correctly in a secure mode in the previous test. From client end, we are testing in the insecure mode because of certs copy blocker on PTFDocker

Reply:
I think we need to test the secure mode because that is the mode we are mostly interested in. btw, I cannot get gnmi_get working in secure mode.

Reply:
Changed to pyclient now.